go/vt/sqlparser/sql.y

```diff
@@ -354,10 +391,449 @@ set_statement:
     $$ = &Set{Comments: Comments($2), Exprs: $3}
   }
 
+column_type:
```

Comment:
It's better to move these at the end. The lose coding standard I've followed is to have to top level statements on top and the lower level ones at the bottom.